src/material-examples/datepicker-disabled/datepicker-disabled-example.ts

```diff
@@ -0,0 +1,20 @@
+import {Component} from '@angular/core';
+import {FormBuilder, FormGroup} from '@angular/forms';
+
+/** @title Disabled datepicker */
+@Component({
+  selector: 'datepicker-disabled-example',
+  templateUrl: 'datepicker-disabled-example.html',
+  styleUrls: ['datepicker-disabled-example.css'],
+})
+export class DatepickerDisabledExample {
+  disabled: FormGroup;
+
+  constructor(fb: FormBuilder) {
+    this.disabled = fb.group({
```

Comment:
Are you using this `FormGroup` anywhere?

Reply:
Oh nope, I was originally gonna do something else for this example and that was leftover